release notes/v0.57.0.md

```diff
@@ -0,0 +1,172 @@
+k6 `v0.57.0` is here 🎉! This release includes:
+
+- Introducing helpers for functional testing.
+- The `k6 new` command now supports templates and ProjectIDs.
+- The `k6/experimental/csv` module gets a new `asObjects` option.
+- We no longer support the `k6/experimental/browser` module.
+- Moving all non-public APIs to the `internal` package.
+
+## Breaking changes
+
+- [#4161](https://github.com/grafana/k6/pull/4161) Drops `k6/experimental/browser`. If you are still using it, please follow the [instructions](https://grafana.com/docs/k6/latest/using-k6-browser/migrating-to-k6-v0-52/) to move to the not experimental module.
+- [#4133](https://github.com/grafana/k6/pull/4133) Moves all not publicly used API in `internal` package. This was based on the publicly available extensions for k6. And may break private ones. There is a good chance more APIs will be removed or changed in follow-up releases after this more mechanical change.
+- [#4292](https://github.com/grafana/k6/pull/4292) TypeScript is automatically supported and recognized if the script files use the `.ts` extension. Consequentially, it deprecates `experimental_enhanced` compatibility-mode as it is no longer necessary.
+
+## New features
+
+### New functional testing focused official jslib [k6-testing](https://github.com/grafana/k6-jslib-testing)
+
+The k6 team has been developing a new official jslib dedicated to functional testing. While it is still under active development and will potentially see breaking changes, the set of APIs and behaviors it offers are meant to make their way into k6 eventually, and it is now available for early feedback.
+
+[k6-testing](https://github.com/grafana/k6-jslib-testing) is a k6 javascript library that offers a seamless way to write functional tests in k6, using a Playwright-compatible assertions API. It exposes an `expect` function, with which assertions can be performed using specific matchers that reflect the expectation.
+When assertions fail, the test will immediately fail with a clear error message, including the expected and actual values in a similar fashion to what users would observe when using Playwright assertions. 
+
+There are many generic matchers (and more to come), such as `toEqual`, `toBe`, or `toBeTruthy`, to only name a few, that can be used to assert conditions during a k6 test.
```

Comment:
It's worth pointing to the documentation, e.g. this GitHub README or a better place

```suggestion
[There are many generic matchers](https://github.com/grafana/k6-jslib-testing?tab=readme-ov-file#3-standard-assertions) (and more to come), such as `toEqual`, `toBe`, or `toBeTruthy`, to only name a few, that can be used to assert conditions during a k6 test.
```